include/math/pi.hpp

```diff
@@ -0,0 +1,9 @@
+#ifndef _RIVE_PI_CONSTANT_HPP_
+#define _RIVE_PI_CONSTANT_HPP_
+
+namespace rive
+{
+    constexpr float pi = 3.14159265358979323846;
```

Comment:
Not sure if this is the best place for this constant

Reply:
Np, but you can use M_PI from cmath:

```
#include <cmath>
// now M_PI is defined.
```

Reply:
removed this file and using M_PI now